### PR TITLE
Malware Sandboxes PR

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/44495
fixes: https://github.com/demisto/etc/issues/42403
fixes: https://github.com/demisto/etc/issues/44496

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by David-BMS** [comment]
@julieschwartz18 All the sandbox content moved to this PR
If no further changes are needed please approve

**Comment by julieschwartz18** [comment]
Doc review complete

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_and_Analyze_File_-_Generic.yml [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: e64d0590-3e6c-4b99-8ded-da01909d2195
+    type: condition
+    task:
+      id: e64d0590-3e6c-4b99-8ded-da01909d2195
+      version: -1
+      name: Is Wildfile enabled?
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_and_Analyze_File_-_Generic.yml:245]: wildfire

File: Packs/CommonPlaybooks/Playbooks/playbook-Search_For_Hash_In_Sandbox_-_Generic.yml [hunk 2]
```diff
@@ -0,0 +1,722 @@
...
+      {
+        "position": {
+          "x": -250,
+          "y": 1170
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Search_For_Hash_In_Sandbox_-_Generic.yml:390]: should be true no ?

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_and_Analyze_File_-_Generic.yml [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+      {
+        "position": {
+          "x": 80,
+          "y": 1080
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_and_Analyze_File_-_Generic.yml:375]: should be true no?

File: Packs/CommonPlaybooks/ReleaseNotes/2_2_2.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#### Playbooks
+##### New: Search For Hash In Sandbox - Generic
+This playbook searches for a specific hash in the supported sandboxes. If the hash is known, the playbook provides a detailed analysis of the sandbox report. Currently supported sandboxes are Falcon X and Wildfire.  (Available from Cortex XSOAR 6.5.0).
+##### New: Detonate and Analyze File - Generic
+This playbook uploads, detonates, and analyzes files for the supported sandboxes. Currently supported sandboxes are Falcon X and Wildfire.
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/ReleaseNotes/2_2_2.md:5]: Should add (Available from Cortex XSOAR 6.5.0)

File: Packs/CrowdStrikeFalconX/Playbooks/playbook-FalconX_Detonate_and_Analyze_File.yml [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+              simple: Windows,Linux,Android
+          mapped_values:
+            value:
+              simple: '160: Windows 10,300: Linux Ubuntu 16.04,300: Linux Ubuntu 16.04'
+  required: false
+  description: |-
+    The operating system for which the alert was raised.
+    accepted values are Windows, Linux, Android
+
+    ..
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalconX/Playbooks/playbook-FalconX_Detonate_and_Analyze_File.yml:373]: This dots should be here ?

**Review by ssokolovich — APPROVED** [review]
Please check those comments before merging

**Pull request merged**